selftests/job_api/test_features.py

```diff
@@ -84,20 +90,35 @@ def workdir_file_path(self):
         file_name = self.params.get('file')
         return os.path.join(self.latest_workdir, file_name)
 
+    def test_check_archive_file_exists(self):
```

Comment:
I understand this check could *almost* reuse the "Test if a file was created", but the dynamic aspect of the logdir makes it hard. So this LGTM.

Reply:
Right, tests that depend on variables from the job are tricky. I'm struggling while trying to merge tests as much as I can.